src/subcommand/wallet/pending.rs

```diff
@@ -0,0 +1,28 @@
+use super::*;
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct PendingOutput {
+  pub rune: SpacedRune,
+  pub commit: Txid,
+}
+#[derive(Debug, Parser)]
+pub(crate) struct Pending {}
+
+impl Pending {
+  pub(crate) fn run(self, wallet: Wallet) -> SubcommandResult {
+    let etchings = wallet
+      .pending_etchings()?
+      .into_iter()
+      .map(|(rune, entry)| {
+        let spaced_rune = rune.to_string().parse::<SpacedRune>().unwrap();
```

Comment:
@raphjaph what's the best way to get a `SpacedRune` from a `Rune`?

Reply:
It's buried inside the entry struct. Something like this:
```
let spaced_rune = entry.output.rune.unwrap().rune
```